fission/flake.nix

```
@@ -0,0 +1,42 @@
+{
+  description = "Synthesis' Web-Based Robot Simulator";
+
+  inputs.nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+
+  outputs =
+    inputs:
+    let
+      supportedSystems = [
+        "x86_64-linux"
+        "aarch64-linux"
+        "x86_64-darwin"
+        "aarch64-darwin"
+      ];
+      forEachSupportedSystem =
+        f:
+        inputs.nixpkgs.lib.genAttrs supportedSystems (
+          system:
+          f {
+            pkgs = import inputs.nixpkgs { inherit system; };
+          }
+        );
+    in
+    {
+      devShells = forEachSupportedSystem (
+        { pkgs }:
+        {
+          default = pkgs.mkShell {
+            nativeBuildInputs = with pkgs; [
+              playwright-driver.browsers
+            ];
+
+            shellHook = ''
+              export PLAYWRIGHT_BROWSERS_PATH=${pkgs.playwright-driver.browsers}
+              export PLAYWRIGHT_SKIP_VALIDATE_HOST_REQUIREMENTS=true
+            '';
+
+          };
+        }
+      );
+    };
```

would be good to add a formatter for nix code
```suggestion
      formatter = forEachSupportedSystem (pkgs: pkgs.nixfmt-tree);
    };
```